Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:wheezy
+MAINTAINER Di Wu <diwu@yelp.com>
+
+COPY . /opt/redash/current/
+
+# Install dependencies
+WORKDIR /opt/redash/current
+RUN bash setup/bootstrap_docker.sh
+
+# Build frontend sources
+WORKDIR /opt/redash/current/rd_ui
+USER redash
+RUN mkdir -p /opt/redash/.npm/node_modules
+RUN npm config set prefix /opt/redash/.npm/node_modules
+RUN npm install -g bower
+RUN npm install -g grunt-cli
+RUN npm install
+RUN /opt/redash/.npm/node_modules/bin/bower install
+RUN /opt/redash/.npm/node_modules/bin/grunt build
```

Comment:
The whole Node.js stack isn't needed in production (the release tarballs already include "compiled" assets), and shouldn't be part of the base image. We can create another image for development, that derives from the base one and installs the Node.js stack.